src/components/scenegraph/Toolbar.js

```diff
@@ -198,6 +207,11 @@ export default class Toolbar extends Component {
         filteredData.version
       );
 
+      const isImagePathEmpty = await checkIfImagePathIsEmpty(currentSceneId);
+
+      if (!doSaveAs && isImagePathEmpty && !doSaveAs && isImagePathEmpty) {
```

Comment:
this appears to be redundant, instead why not:

`if (!doSaveAs && isImagePathEmpty) {`